dash/development/component_generator.py

```diff
@@ -83,10 +115,16 @@ def cli():
         default='package.json',
         help='The filename of the copied `package.json` to `project_shortname`'
     )
+    parser.add_argument(
+        '-r', '--rlang',
+        action='store_true',
+        help='Generate Dash components for R, and package for installation.'
```

Comment:
Add a bit more depth to the help, like where the R files will be generated. Also add that it's experimental at the moment.